cegs_portal/search/templates/search/v1/search_results.html

```diff
@@ -294,11 +453,24 @@
 
         updateFacetState();
 
+        let searchFeatures = queryParams.getAll("feature_type");
+        featureCheckboxes.forEach(checkbox => {
+            if (checkbox.id !== 'Exon' && checkbox.id !== 'Transcript') {
+                checkbox.checked = true;
+            }
```

Comment:
This is close, but ignores potential `feature_type` parameters in the URL. 

```suggestion
           // If there are any feature_type query parameters use those to determine what is checked.
           // Otherwise, use the defaults (everything but exons and transcripts)
            if (searchFeatures.size != 0) {
                checkbox.checked = searchFeatures.includes(checkbox.id);
            } else if (checkbox.id !== 'Exon' && checkbox.id !== 'Transcript') {
                checkbox.checked = true;
            }
```

Reply:
This seems to nullify the initial if statement. Ill see what I can do

Reply:
What do you mean nullify?

Reply:
When I load the page, the default feature_types aren't checked anymore and the table displays all feature_types
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/1697d476-bbe1-4a40-9eb0-e4a6b893d34b)

Reply:
oh, it should be .length, not .size

Reply:
Right! Not native to JS